routes/manual.tsx

```diff
@@ -356,8 +356,9 @@ export default function Manual({ params, url, data }: PageProps<Data>) {
               <div class={tw`pt-1`}>
                 <Markdown
                   source={data.content
-                    .replace(/\$STD_VERSION/g, stdVersion)
-                    .replace(/\$CLI_VERSION/g, version)}
+                    .replaceAll(".md", "")
```

Comment:
this doesnt seem ideal; what if we mention .md files in some manual entry? I think we need to figure out why the URLs have .md in the first place, and remove that

Reply:
> what if we mention .md files in some manual entry?

We can make the regex only match markdown links (`[text](link)`) and that should avoid most false positives.

> I think we need to figure out why the URLs have .md in the first place, and remove that

If the file extension is removed, the link becomes invalid in the _source_ file. I think it's better to do transformation here.